lib/asar/fs-wrapper.ts

```diff
@@ -37,6 +37,8 @@ const getOrCreateArchive = (archivePath: string) => {
   return newArchive;
 };
 
+const asarRe = /\.asar/i;
```

Comment:
nit: since this is only used inside `splitPath` it may be slightly better to declare it in that scope but it's not really a significant different so i'm fine either way :)

Reply:
I thought about it but in the snipped I posted in my first post having the regex inlined made the checks take about 50% more time (~90ms vs ~60ms) on my machine.

Having it inlined would be cleaner though.

Either way would work for me, let's go with whatever you prefer.